Controls/LoadablePanel/src/DotVVM.Contrib.Samples/ViewModels/PagingRepeaterViewModel.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DotVVM.Framework.Controls;
+using DotVVM.Framework.ViewModel;
+
+namespace DotVVM.Contrib.Samples.ViewModels
+{
+    public class PagingRepeaterViewModel : MasterViewModel
+    {
+        public GridViewDataSet<Item> Grid { get; set; } = new GridViewDataSet<Item>
+        {
+            PagingOptions = new PagingOptions
+            {
+                PageSize = 6,
+                TotalItemsCount = 32
+            }
+        };
+
+        public List<string> LoadingItems { get; set; } = new List<string>();
+
+
+        [AllowStaticCommand]
+        public async Task<string> LoadItem(string id)
```

Comment:
This is really bad practice. You have 2 options. 
A) Use static methods in ViewModel with AllowStatiCommand. 
B) Use instantiation methods in static command service. (just separate class, it does not have any ancestor) 

But please do not mix this. We are preparing a Roslyn analyzer to detect this pattern and prevent people from using it.

Reply:
I reworked the sample project structure to comply with latest DotVVM best practices.